src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.netcoreapp.cs

```diff
@@ -2,18 +2,34 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
+using System.Runtime.InteropServices;
 using System.Security.AccessControl;
+using Microsoft.Win32.SafeHandles;
 
 namespace System.IO
 {
     public static partial class FileSystemAclExtensions
     {
+        #region Private members
+
+        private const int GENERIC_READ = unchecked((int)0x80000000);
+        private const int GENERIC_WRITE = 0x40000000;
+
+        #endregion
+
+        #region Public methods
+
         /// <summary>Creates a new directory, ensuring it is created with the specified directory security. If the directory already exists, nothing is done.</summary>
         /// <param name="directoryInfo">The object describing a directory that does not exist in disk yet.</param>
         /// <param name="directorySecurity">An object that determines the access control and audit security for the directory.</param>
         /// <exception cref="ArgumentNullException"><paramref name="directoryInfo" /> or <paramref name="directorySecurity" /> is <see langword="null" />.</exception>
         /// <exception cref="DirectoryNotFoundException">Could not find a part of the path.</exception>
         /// <exception cref="UnauthorizedAccessException">Access to the path is denied.</exception>
+        /// <remarks><format type="text/markdown"><![CDATA[
+        ///## Remarks
+        ///This extension method was added to .NET Core to bring the functionality that was provided by the <xref:System.IO.DirectoryInfo.Create(System.Security.AccessControl.DirectorySecurity)> .NET Framework method.
+        /// ]]></format></remarks>
```

Comment:
I don't think we should be using this custom XML syntax in our doc comments.  Any such formatting should happen as part of bringing the comments into the docs system.

Reply:
I removed the markdown, although it's allowed (no build error).
Note: By removing the markdown, The .NET Framework method I mention in the remark cannot be resolved by Visual Studio, so I will not be adding it as a `<see cref="System.IO.DirectoryInfo.Create(System.Security.AccessControl.DirectorySecurity)" />`.